frontend/src/interface/processBackendData.js

```diff
@@ -72,3 +74,40 @@ export function processBackendData(data, callback) {
 
     callback();
 }
+/**
+ *
+ * @param {*} projectDescription The comments mentioned in SimaPro Appliction.
+ * It if of the form "Categorie:[Generation/Transmission/IndustrialApplication]/[Solutions/Product/Services]#Model:ModelId#Location:loaction""
+ * This function segrigates the projects based on Generation, Transmission and Industrial APplications as mentioned in peojectDescription.
+ * @returns projects based on Generation, Transmission and Industrial APplications
+ */
+export function projectCategoryProcessing(projectDescription) {
+    if (
+        projectDescription === [] ||
+        projectDescription === undefined ||
+        projectDescription === null
+    ) {
+        return [categories.industrialApplications];
```

Comment:
why if null do we return exactly industrial applications?

Reply:
@SaiVarunVaranasi do you know? I am unsure. But if we change that, we need to handle `[]` cases I guess.

Reply:
I changed that ;)